README.md

```diff
@@ -71,13 +74,20 @@ One of the `[MASK_EXTRA]` dependencies is the `lightgbm` package. On Windows it
 
 ### PyPi distribution
 
-The package requires Python version **>=3.8** . It can be installed with:
+The package requires Python version **>=3.8** . If this is your first time installing `eo-learn`, or you aren't sure what kind of installation you require, than feel free to install it with:
+
+```bash
+pip install "eo-learn[ALL]"
+```
+
+This will install the package and all the default modules. Otherwise, if you're interested in a super light, core-only installation of `eo-learn`, then install it via:
 
 ```bash
 pip install eo-learn
+# pip install "eo-learn[CORE]"  # this does the same thing
```

Comment:
actually it does not because there is no `[CORE]` extra :)

Reply:
fixed